test/test_helper.rb

```diff
@@ -10,7 +10,7 @@
 
 require "minitest/autorun"
 require "minitest/reporters"
-require "mocha/setup"
+require "mocha/minitest"
```

Comment:
Is this change directly related to the warning issues?

Reply:
It's related to a separate warning that came up while I was running the test suite locally. The mocha gem tells you that "mocha/setup" is deprecated, and that you should require one of the more specific setups. This is the appropriate one for your test suite.